blog/_src/posts/2017-08-11-on-being-a-pc-member-rebuttals-and-responses.md

```diff
@@ -0,0 +1,104 @@
+strongly be avoided -- we should even stop giving that advice.
+
+<!-- more -->
+
+This year, I had the honor (and accompanying load of work) of being
+a Program Committee (PC) member at the ICFP conference. It was first
+time being a PC member at a conference, and I found it extremely
+pleasant and interesting, thanks to the authors who sent us articles
+to review, my fellow PC members, and the ever-smiling careful
+balancing work of our PC chair, Mark Jones. It was also a lot of work,
+starting with 18 reviews to do over a month and a half, an intense PC
+meeting, and the new "second phase" process with the opportunity for
+authors and reviewers to exchange feedback on changes requested by the
+program committee.
+
+There is little guidance on how to write author responses, although
+this [blog
+post](http://www.pl-enthusiast.net/2014/09/17/advice-writing-author-response/)
+by Michael Hicks on pl-enthusiast is quite good. One thing that is
+obvious as a reviewer and is only slightly brushed in this post,
+however, is that author responses should *not* aim to fit a 500 words
+limit, and in fact I believe that it is a very bad idea to do so.
+
+As for most conference, the ICFP review system recommends (in writing)
+to authors to keep their response to 500 words (some systems also
+highlight words after those in red to make the point clear). Don't do
+this! The least convincing responses I have seen are those that
+followed this recommendation.
+
+We have a frustrating situation where the explicit rule is different
+from the thing people do in practice. This is bad for newcomers that
+do not know the norms and cannot tell if ignoring the rule may hurt
+them. This is the point of this blog post:
+
+- If you are an author, please know that disrespecting the 500-words
+  limit is the *right thing* to do. You should also know, of course,
+  that people have limited time, so keep the main body of your
+  response reasonably long. (I spent a day and a half on your paper
+  already, I am willing to spend 10 additional minutes reading the
+  response.)
+
+- If you are a program committee chair or a Magical HotCRP Wizard,
+  please remove this silly recommendation to keep responses to 500
+  words.
+
+### My personal response format
+
+My author responses start with general comments that elaborate on the
+main point I want to tell all reviewers. Then, a second section
+contains per-reviewer comments (in as many subsections as reviewers);
+it is clearly marked as skippable. Here is the skeleton of the last
+response I wrote:
+
+> We thank the reviewers for their work on our article and their
+> detailed feedback.  We start with a general discussion that responds
+> to the salient points raised by reviewers.  In a second part, we
+> provide detailed responses to the questions/remarks of each
+> reviewer.
+>
+> ## General discussion
+>
+> [..]
+>
+> ## Specific questions/comments: reviewer #A
```

Comment:
(From Matthias via me:) write "review `#A`" instead of "reviewer ...", because you're replying just to the words in the review and not the human being that wrote it.